libs/shared/ui/src/lib/components/error-boundary/error-boundary.tsx

```diff
@@ -25,15 +24,11 @@ export function ErrorFallback({ className, error, resetErrorBoundary }: Fallback
   )
 }
 
-export function ErrorBoundary({ children }: PropsWithChildren) {
-  const location = useLocation()
+interface ErrorBoundaryProps extends PropsWithChildren, Attributes {}
 
+export function ErrorBoundary({ key, children }: ErrorBoundaryProps) {
   return (
-    <ErrorBoundaryRaw
-      key={location.pathname}
-      FallbackComponent={ErrorFallback}
-      onReset={() => window.location.reload()}
-    >
+    <ErrorBoundaryRaw key={key} FallbackComponent={ErrorFallback} onReset={() => window.location.reload()}>
```

Comment:
key is really necessary ? I have a warning in the console, maybe related
 
<img width="542" alt="Screenshot 2024-09-11 at 17 15 27" src="https://github.com/user-attachments/assets/64544e1d-d2b5-4e6e-b3c4-3dd1c18a4816">

Reply:
Fixed https://github.com/Qovery/console/pull/1660/commits/5f279de42063801fccbd75a76200084581fc3c89